scaffolding-chef-inspec/tests/linux/user-a2-profile-dep/habitat/plan.sh

```diff
@@ -0,0 +1,11 @@
+pkg_name=effortless-a2-profile-test
+pkg_version=0.1.0
+pkg_origin=chef
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_description="The Audit Baseline InSpec Profile"
+pkg_upstream_url="http://chef.io"
```

Comment:
Please remove all metadata from the test package

`pkg_maintainer`
`pkg_license`
`pkg_description`
`pkg_upstream_url`

Removing this helps focus the test, and makes it more readable and understandable for what is under test.